pulpcore/cli/common/__init__.py

```diff
@@ -80,6 +80,7 @@ def _config_callback(ctx: click.Context, param: Any, value: Optional[str]) -> No
 @click.option(
     "--profile",
     "-p",
+    envvar="PULP_PROFILE",
```

Comment:
What do you think about "PULP_CLI_PROFILE"?

Reply:
+1